test/mocks/appium/wdio_driver.js

```diff
@@ -79,6 +79,7 @@ class Browser {
       }
     });
     this.getOrientation = jasmine.createSpy().and.returnValue('PORTRAIT');
+    this.$ = jasmine.createSpy().and.returnValue(Promise.resolve('element'));
```

Comment:
ideally resolve it to an object which is similar to element type that it returns, so that its easy to debug and write an integration test

Reply:
yep resolving it to objs